index_impl.go

```diff
@@ -495,11 +495,13 @@ func (i *indexImpl) SearchInContext(ctx context.Context, req *SearchRequest) (sr
 			err = serr
 		}
 		if sr != nil {
-			sr.BytesRead = totalBytesRead
+			sr.Cost = totalSearchCost
 		}
```

Comment:
Maybe it's just me who feels this way, but I think the previous "BytesRead" was better than Cost. Cost is essentially being added for serverless and I think the concept of serverless should be restricted to cbft. BytesRead is a general enough name. Thoughts?

Reply:
the reason why i changed from bytesRead to cost is because i thought the name cost seems generic enough such that if in future the scale of the value changed (KiB or MiB etc.) or if we decided to return the CPU cycles this search corresponded to rather than bytes read from the mmaped file, the name "cost" would still hold true in that case.

Reply:
Agree  @metonymic-smokey